qlty-plugins/plugins/linters/zizmor/plugin.toml

```diff
@@ -0,0 +1,22 @@
+config_version = "0"
+
+[plugins.definitions.zizmor]
+runtime = "python"
+package = "zizmor"
+file_types = ["github-workflow"]
+latest_version = "1.11.0"
+known_good_version = "1.11.0"
+version_command = "zizmor --version"
+config_files = ["zizmor.yml"]
+description = "Static analysis tool for GitHub Actions"
+security = true
+
+[plugins.definitions.zizmor.drivers.lint]
+script = "zizmor --format=sarif ${target}"
```

Comment:
I think Zizmore has a few different modes like offline vs. online.. I think we should use offline mode by default (faster, rate limits)